### ISSUE TITLE
Outlook add-in preview : missing folder

### ISSUE THREAD

**Issue by michaelmaillot** [SharePoint/sp-dev-docs#5219]
## Category
- [x] Question
- [ ] Typo
- [ ] Additional article idea

## Question

Hi,

I wanted to try the Office Add-In Preview but when creating the WebPart, the "officeAddin" folder is missing (although I'm using the v1.10 of SPFx). Below some screen captures:

![image](https://user-images.githubusercontent.com/44271458/73082795-6199b280-3eca-11ea-9fb2-8b55147adb01.png)

![image](https://user-images.githubusercontent.com/44271458/73083188-0916e500-3ecb-11ea-9d23-c7796adbf15d.png)

Am I missing something regarding the requirements ?

Thank you in advance.

## Environment details

 - OS: Windows 10
 - Framework: Node.js v10.16.0
 - Tooling: VS Code, SPFx v1.10.0

![image](https://user-images.githubusercontent.com/44271458/73082832-74ac8280-3eca-11ea-8937-2ddcd2381089.png)

**Comment by VesaJuvonen** [comment]
You will need to remember to execute the Yeoman generator in preview mode as follows:

```
yo @microsoft/sharepoint --plusbeta
```
Please follow up the needed steps from this tutorial - https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/office-addins-tutorial

**Comment by patmill** [comment]
(Just a note - the --plusbeta also gives you access to the office SDK / API via the webpart interfaces.)

**Comment by michaelmaillot** [comment]
> You will need to remember to execute the Yeoman generator in preview mode as follows:
> 
> ```
> yo @microsoft/sharepoint --plusbeta
> ```
> 
> Please follow up the needed steps from this tutorial - https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/office-addins-tutorial

Thanks @VesaJuvonen, but if you have a closer look in my first screen capture on the top, you'll see that this is exactly the same command that I've set in my PowerShell Interface (I've started to follow your video tutorial btw 😉).

Thanks @patmill for the tip. Is it possible to reopen the issue, as my problem has not been solved yet ?

**Comment by VesaJuvonen** [comment]
hmm - makes no sense what so ever as just tested on two Win 10 computers with expected outcome. Re-installation of SPFx 1.10 globally and then testing that everything works as expected... without any issues.

**Comment by michaelmaillot** [comment]
Ok so the problem was slightly PEBKAC 😅 

I used to have two versions of Node (through NVM) for multi-environment purposes and until now, I had no problem to switch between them and init SPFx projects with their specific versions.

When I had a look at my package.json file, I was in 1.9.1 version and that's why I didn't get the officeAddin folder. Don't know how but even if I was pointing the Node version 10 (which holds the 1.10 version of SPFx), it was the oldest version of SPFx that was targeted (on my old version of Node).

So I had to completely remove @microsoft/generator-sharepoint (on every environment), then reinstall only latest on the most recent version of Node and everything's fine !

Thanks for you help, closing the ticket.
